README.md

```diff
@@ -0,0 +1,112 @@
+
+
+# Cisco NXOS Collection
+[![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/project/github.com/ansible-collections/cisco.nxos) <!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/vyos)](https://codecov.io/gh/ansible-collections/cisco.nxos)-->
+
+The Ansible Cisco NXOS collection includes a variety of Ansible content to help automate the management of Cisco NXOS network appliances.
+
```

Comment:
Can we add

# Content Overview

The goal of the NXOS connection plugin when combined with NXOS resource modules (link here) is to provide an experience for network operators and engineers a familiar experience for NXOS that aligns with the other core networking platform supported by Ansible. Ongoing development efforts and contributions to this collection are solely focused on enhancements to current resource modules, additional resource modules and enhancements to connection plugins.

Reply:
I added the first apart here. The ongoing development part I added to the contributing section.